### PR TITLE
Fixes #66 - () should not be shown in the popup window

### LINKED ISSUE #66: () is shown in alert notification window
link: http://community.mybb.com/thread-169382-post-1160848.html#pid1160848

As @Euan wrote in this post (http://community.mybb.com/thread-169382-post-1161017.html#pid1161017 ), we just need to adjust lang file and users have to use latest MyAlerts version (2.0.1 and higher).

### PR DESCRIPTION
**Pull request by Cu8eR**
Fixes #66 - () should not be shown in the popup window

@WhiteNeo - please could you verify this PR?

### CODE REVIEW

**Comment by euantorano** [comment]
Might be good to remove the `dateline` here too: https://github.com/Cu8eR/thankyou-like-plugin/blob/master/inc/plugins/thankyoulike.php#L1006

**Comment by Cu8eR** [comment]
@euantorano - thank you, now it should be fine :)

**Comment by euantorano** [comment]
@Cu8eR almost, there's an extra comma after the last parameter now ;)

File: inc/plugins/thankyoulike.php [hunk 1]
```diff
@@ -1002,8 +1002,7 @@ public function formatAlert(MybbStuff_MyAlerts_Entity_Alert $alert, array $outpu
 			return $this->lang->sprintf(
 				$this->lang->tyl_alert,
 				$outputAlert['from_user'],
-				htmlspecialchars_uni($alertContent['t_subject']),					
-				$outputAlert['dateline']
+				htmlspecialchars_uni($alertContent['t_subject']),
```
Comment (euantorano) [inline inc/plugins/thankyoulike.php]: Should be: `htmlspecialchars_uni($alertContent['t_subject'])`

**Comment by Cu8eR** [comment]
I really should buy new glasses or use bigger font :D thx @euantorano

**Comment by euantorano** [comment]
:wink: Not a problem!

**Pull request merged**